aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -176,9 +176,16 @@ module aptos_framework::stake {
         mint_cap: MintCapability<AptosCoin>,
     }
 
+    // TODO change all types to u32 ?
+    struct IndividualValidatorPerformance has copy, store, drop {
```

Comment:
This doesn't need copy or drop.

Reply:
we need drop to be able to do:
`validator_perf.validators = vector::empty();`
below